app/Http/Controllers/Apis/Protected/Main/OAuth2SponsoredProjectApiController.php

```diff
@@ -423,4 +426,34 @@ public function getSupportingCompany($id, $sponsorship_type_id, $company_id){
             return $sponsorship_type->getSupportingCompanyById(intval($company_id));
         }, $company_id);
     }
+
+    /**
+     * @param LaravelRequest $request
+     * @param $project_id
+     * @return mixed
+     */
+    public function addSponsoredProjectLogo(LaravelRequest $request, $project_id)
+    {
+        return $this->processRequest(function () use ($request, $project_id) {
+            $file = $request->file('file');
+            if (is_null($file)) {
+                return $this->error412(array('file param not set!'));
```

Comment:
this is not needed ( dont enclose on array ) error412 already does

Reply:
done